installing/installing_aws/preparing-to-install-on-aws.adoc

```diff
@@ -21,7 +21,14 @@ If the cloud identity and access management (IAM) APIs are not accessible in you
 [id="choosing-an-method-to-install-ocp-on-aws"]
 == Choosing a method to install {product-title} on AWS
 
-You can install {product-title} on AWS by using one of the following deployment methods.
+You can install {product-title} on installer-provisioned or user-provisioned infrastructure. The default installation type uses installer-provisioned infrastructure, where the installation program provisions the underlying infrastructure for the cluster. You can also install {product-title} on infrastructure that you provision. If you do not use infrastructure that the installation program provisions, you must manage and maintain the cluster resources yourself.
+
+See xref:../../architecture/architecture-installation.adoc#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+
+[id="choosing-an-method-to-install-ocp-on-aws-installer-provisioned"]
+=== Installing a cluster on installer-provisioned infrastructure
+
+You can install a cluster on AWS infrastructure that is provisioned by the {product-title} installation program, by using one of the following methods.
```

Comment:
I'd suggest using a colon to end each of these sentences that leads into the bulleted lists.

Reply:
@bergerhoffer thank you for your review and for this suggestion. I have now resolved this in each of the assemblies.